yarn.lock

```diff
@@ -10691,17 +10696,18 @@ electron-to-chromium@^1.3.712:
   integrity sha512-3kRVibBeCM4vsgoHHGKHmPocLqtFAGTrebXxxtgKs87hNUzXrX2NuS3jnBys7IozCnw7viQlozxKkmty2KNfrw==
 
 electron-updater@^4.3.5:
-  version "4.3.5"
-  resolved "https://registry.yarnpkg.com/electron-updater/-/electron-updater-4.3.5.tgz#4fb36f593a031c87ea07ee141c9f064d5deffb15"
-  integrity sha512-5jjN7ebvfj1cLI0VZMdCnJk6aC4bP+dy7ryBf21vArR0JzpRVk0OZHA2QBD+H5rm6ZSeDYHOY6+8PrMEqJ4wlQ==
-  dependencies:
-    "@types/semver" "^7.3.1"
-    builder-util-runtime "8.7.2"
-    fs-extra "^9.0.1"
-    js-yaml "^3.14.0"
+  version "4.3.9"
```

Comment:
@tomasklim Had a good point on Slack: If we want bump the package to 4.3.9 why don't we change the version explicitly in `package.json` instead of just running `yarn upgrade electron-updater` and relying on the caret range?

TBH, I'm not sure. My thought process was to keep the diff minimal and 'take advantage' of the range. Tomas, on the other hand, expects the `package.json` to be explicit about what version do we require.

What do you think @matejkriz ?

Reply:
TBH, I think it really doesn't matter – whenever I want to check the version of some package, I check yarn.lock or node_modules folder... And when I want to upgrade some package, I don't care much about the range in package.json, I always look on changelog of that package and select the desired version from there.

So both ways seem perfectly fine to me.

Reply:
Allright. I'll resolve and merge.